TTS/tts/models/xtts.py

```diff
@@ -311,7 +312,7 @@ def __init__(self, config: Coqpit):
     def init_models(self):
         """Initialize the models. We do it here since we need to load the tokenizer first."""
         if self.tokenizer.tokenizer is not None:
-            self.args.gpt_number_text_tokens = self.tokenizer.tokenizer.get_vocab_size()
+            self.args.gpt_number_text_tokens = max(self.tokenizer.tokenizer.get_vocab().values()) + 1
```

Comment:
How about adding it as a tokenizer method? like self.tokenizer.get_vocab_size()

Reply:
`get_vocab_size` is already meaning something else to the tokenizer, but yeah, let's do `get_number_tokens`